xchange-examples/src/main/java/org/knowm/xchange/examples/blockchain/account/BlockchainAccountDemo.java

```diff
@@ -0,0 +1,91 @@
+package org.knowm.xchange.examples.blockchain.account;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.ExchangeFactory;
+import org.knowm.xchange.blockchain.BlockchainExchange;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.account.AccountInfo;
+import org.knowm.xchange.dto.account.AddressWithTag;
+import org.knowm.xchange.dto.account.Fee;
+import org.knowm.xchange.dto.account.FundingRecord;
+import org.knowm.xchange.instrument.Instrument;
+import org.knowm.xchange.service.account.AccountService;
+import org.knowm.xchange.service.trade.params.HistoryParamsFundingType;
+import org.knowm.xchange.service.trade.params.TradeHistoryParams;
+import org.knowm.xchange.service.trade.params.TradeHistoryParamsTimeSpan;
+
+import static org.knowm.xchange.examples.blockchain.BlockchainDemoUtils.END_TIME;
+
+/** @author scuevas*/
+public class BlockchainAccountDemo {
+
+  private static final String API_KEY = "";
+  private static final String API_SECRET = "";
+  private static final Exchange BLOCKCHAIN_EXCHANGE = ExchangeFactory.INSTANCE.createExchange(BlockchainExchange.class, API_KEY, API_SECRET);
+  private static final ObjectMapper objectMapper = new ObjectMapper().enable(SerializationFeature.INDENT_OUTPUT);
+
+  public static void main(String[] args) throws IOException {
+    System.out.println("===== ACCOUNT SERVICE =====");
+    accountServiceDemo();
+  }
+
+  private static void accountServiceDemo() throws IOException {
+    AccountService accountService = BLOCKCHAIN_EXCHANGE.getAccountService();
+
+    System.out.println("===== getAccountInfo =====");
+    AccountInfo accountInfo = accountService.getAccountInfo();
+    System.out.println(objectMapper.writeValueAsString(accountInfo));
+
+    System.out.println("===== withdrawFunds =====");
+    String withdraw = accountService.withdrawFunds(Currency.BTC, BigDecimal.valueOf(0.00005), "ea1f34b3-e77a-4646-9cfa-5d6d3518c6d3");
+    System.out.println(objectMapper.writeValueAsString(withdraw));
+
+    System.out.println("===== requestDepositAddress =====");
+    String address = accountService.requestDepositAddress(Currency.ETH);
+    System.out.println(objectMapper.writeValueAsString(address));
+
+    System.out.println("===== requestDepositAddressData =====");
+    AddressWithTag addressWithTag = accountService.requestDepositAddressData(Currency.ETH);
+    System.out.println(objectMapper.writeValueAsString(addressWithTag));
+
+    System.out.println("===== getFundingHistory =====");
+    TradeHistoryParams params = accountService.createFundingHistoryParams();
+
+    if (params instanceof TradeHistoryParamsTimeSpan) {
```

Comment:
if I am not wrong you do not need to do this if, this condition is always true. Just remove it